aspnet/client-side/bundling-and-minification.rst

```diff
@@ -1,8 +1,315 @@
-.. include:: /../common/stub-topic.txt
+Bundling and Minification
+=========================
+
+By `Erik Reitan`_ 
+
+Bundling and minification are two techniques you can use in ASP.NET 5 to improve page load performance for your web application. Bundling makes it easy to combine or bundle multiple files into a single file. Minification performs a variety of different code optimizations to scripts and CSS, which results in smaller file sizes.
+
```

Comment:
Prefer my original intro
Bundling and minification are two techniques you can use in ASP.NET 4.5 to improve request load time.  Bundling and minification improves load time by reducing the number of requests to the server and reducing the size of requested assets (such as CSS and JavaScript.)